assets/sass/v2/_identify.scss

```diff
@@ -78,3 +78,20 @@
     }
   }
 }
+
+.o-form-button-bar {
+  .links-container {
```

Comment:
seems its not yet updated ?

Reply:
good catch! I had this change locally but not committed. It's fixed now.